src/middlewared/middlewared/api/v25_04_0/common.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, List, Optional
+
+from middlewared.api.base import *
+
+__all__ = ["QueryOptions", "QueryArgs"]
+
+
+class QueryOptions(BaseModel):
+    relationships: bool = True
+    extend: Optional[str] = None
```

Comment:
We've created a NEP for defining rules about how we do type annotations. I do not like using the `types` or `typing` module unless absolutely necessary. Instead of doing `Optional[str]`, we can replace it by doing `extend: str | None = None`